tests/lib/snaps/test-snapd-service-watchdog/meta/snap.yaml

```diff
@@ -0,0 +1,15 @@
+name: test-snapd-service-watchdog
+version: 1.0
+apps:
+  direct-watchdog-ok:
+    command: bin/direct
+    daemon: simple
+    watchdog-timeout: 1s
+    restart-condition: never
+    plugs: [service-watchdog]
```

Comment:
It is a little awkward that for `watchdog-timeout` to work, the snap developer much specify it *and* the plugs. We've had related conversations on a command definition implying interfaces or an interface attribute implying a unit directive. IIRC, these were discussed wrt dbus session services and the discussion went unresolved.

If this is the approved design, that's fine, it it isn't, perhaps it is worth discussing if defining two things in yaml to make one thing work is how this should work.

Reply:
This is a good point. The one advantage of this design is that we have fine-grained control about what can use interfaces and where, while once we have a watchdog-timeout, we either accept the snap everywhere, or we don't take it in the store at all.

We'll discuss this further.